README.md

```diff
@@ -36,13 +37,16 @@ For more on Mongoid support in ActiveAdmin see [this issue](https://github.com/g
 Add the following gems to your application's Gemfile, and lock the version:
 
 ```ruby
-gem 'activeadmin-mongoid', '0.7.0'
+gem 'activeadmin-mongoid', '0.8.0'
+gem 'ransack-mongoid', github: 'activerecord-hackery/ransack-mongoid'
 ```
 
 You can safely remove the following lines, since are already activeadmin-mongoid dependencies:
 
 ```ruby
 gem 'activeadmin'
+gem 'kaminari'
+gem 'kaminari-mongoid'
```

Comment:
I would not add this here. Even if this is the current state of things, an application might be using these gems directly for the frontend, for example. If we recommend this, and `activeadmin` happened to switch its pagination engine, the application would break.

Reply:
sounds legit. okay, i'll remove it then